Appraisals

```diff
@@ -583,7 +583,7 @@ elsif Gem::Version.new('2.3.0') <= Gem::Version.new(RUBY_VERSION) \
       gem 'excon'
       gem 'faraday'
       gem 'grape'
-      gem 'graphql', '< 1.12.0' # TODO: Pending ddtrace support for breaking changes in >= 1.12.0
+      gem 'graphql', '>= 1.12.0'
```

Comment:
I've bound the minimum version to the latest release as no other gem depends on `graphql` as limiting the lower version bound improves `bundler` dependency resolution time.

Given `graphql`, a gem that has no dependencies, will never resolve to a version prior to `1.12.0' it's safe to set a lower bound this way.